documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1753,6 +1756,18 @@ Set this property to one of the following values:
 To support {link-prefix}:{link-signalling}#[{prodname} signaling], if you set `capture.scope` to `database`, the xref:mongodb-property-signal-data-collection[signaling data collection] must reside in a database that is specified by the xref:mongodb-property-capture-target[`capture.target`] property.
 ====
 
+`collection`:: Opens a change stream cursor for a single collection to watch for changes to that collection.
++
+[NOTE]
+====
+This feature is currently in incubating state, i.e. exact semantics, configuration options etc. may change in future revisions, based on the feedback we receive.
+====
++
+[WARNING]
+====
+When `capture.scope` is set to `collection`, the connector does not support {link-prefix}:{link-signalling}#[{prodname} signaling] via the `source` channel which is enabled by default. Due to this limitation the connector also does not support incremental snapshots.
```

Comment:
```suggestion
When you set the value of the `capture.scope` property to `collection`, the connector cannot process signals sent through the default `source` {link-prefix}:{link-signalling}[signaling] channel.
Because the `source` channel is required for processing incremental snapshot signals received over any channel, including the Kafka, JMX, and file channels, this limitation prevents the connector from performing incremental snapshots.
```